setup.py

```diff
@@ -10,7 +10,7 @@
     author_email='edward.fitzgerald@fetch.ai',
     packages=find_packages(exclude=['contrib', 'docs', 'tests', 'scripts']),
     package_data={'fetchai': ['ledger/parser/etch.grammar']},
-    install_requires=['requests', 'ecdsa', 'msgpack', 'base58', 'lark-parser', 'semver'],
+    install_requires=['requests', 'ecdsa', 'msgpack', 'base58', 'lark-parser', 'semver', 'pyaes'],
```

Comment:
Does `pyaes` need any native components?

Reply:
No, that's why it was chosen over pycryptodome, it's a pure-python implementation of AES.